types/emoji-mart/dist-es/utils/shared-props.d.ts

```diff
@@ -92,4 +92,7 @@ export interface PickerProps {
     notFound?(): React.Component;
     notFoundEmoji?: string;
     icons?: CustomIcons;
+    darkMode?: boolean;
```

Comment:
wasn't that removed? (obsolated):
https://github.com/missive/emoji-mart/commit/d1138c00e8d2e79c990ea75b89aaea4c7e819c7b